bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/GC.java

```diff
@@ -4323,8 +4323,8 @@ public void setClipping (Path path) {
 	if (path != null) {
 		initGdip();
 		int mode = OS.GetPolyFillMode(handle) == OS.WINDING ? Gdip.FillModeWinding : Gdip.FillModeAlternate;
-		Gdip.GraphicsPath_SetFillMode(path.handle, mode);
-		Gdip.Graphics_SetClipPath(data.gdipGraphics, path.handle);
+		Gdip.GraphicsPath_SetFillMode(Path.win32_getHandle(path, getZoom()), mode);
```

Comment:
extract the two Path.win32_getHandle(path, getZoom()) calls into a variables

Reply:
yessir